docs/INSTALLATION.md

```diff
@@ -0,0 +1,265 @@
+# Installing OVE
+
+OVE needs to be installed before it can be used to control a display. OVE can be installed either by downloading and compiling the source code of the corresponding components or by running a specific installer available on the [**OVE Install**](https://github.com/ove/ove-install/) repository.
+
+All contributors to OVE are encouraged to download and compile the source code. All users of OVE are encouraged to use the OVE installers.
+
+## Installation by running OVE installers
```

Comment:
I would prefer
## Installation using Docker

These are step by step instructions of the quick installer from  [**OVE Install**](https://github.com/ove/ove-install/)

Reply:
I have explained this to @jamesscottbrown as well. Though currently the installer uses Docker compose that targets a Docker standalone installation, this may change in the future, and we need to keep top-level titles generic - this is why this has been kept as such. The instructions are detaled and therefore, I see no issues in having this title.